ui/imports/utils/Utils.qml

```diff
@@ -644,6 +644,44 @@ QtObject {
         return getEtherscanUrl(networkShortName, testnetMode, tx, false /* is TX */)
     }
 
+    // Get Chain Explorer name
+    function getChainExplorerName(networkShortName) {
+        if (networkShortName === Constants.networkShortChainNames.arbitrum) {
+            return qsTr("Arbiscan")
+        }
+        if (networkShortName === Constants.networkShortChainNames.optimism) {
+            return qsTr("Optimistic")
```

Comment:
```suggestion
            return qsTr("Optimism")
```

Reply:
<img width="418" alt="image" src="https://github.com/user-attachments/assets/c7f0f527-13ce-4552-abc0-1dd964ab4bb9">
Figma says this is called Optimistic :D

Reply:
Yeah it is correct